### ISSUE TITLE
Cannot find module 'babel-runtime/helpers/toConsumableArray'

### ISSUE THREAD

**Issue by gdetrez** [webpack-contrib/karma-webpack#311]
**I'm submitting a bug report**

This seems to be a regression introduced in 2.0.11 and still present in 2.0.12:
```sh
> karma start --single-run
27 02 2018 16:01:18.532:ERROR [plugin]: Error during loading "/home/travis/build/gdetrez/karma-webpack-issue/node_modules/karma-webpack" plugin:
  Cannot find module 'babel-runtime/helpers/toConsumableArray'
27 02 2018 16:01:18.536:ERROR [preprocess]: Can not load "webpack", it is not registered!
  Perhaps you are missing some plugin?
27 02 2018 16:01:18.584:INFO [karma]: Karma v1.7.1 server started at http://0.0.0.0:9876/
27 02 2018 16:01:18.584:INFO [launcher]: Launching browser Chrome with unlimited concurrency
27 02 2018 16:01:18.585:ERROR [karma]: Found 2 load errors
```

I have a travis job that shows the problem:
https://travis-ci.org/gdetrez/karma-webpack-issue/builds/346843566

Here are the sources:
https://github.com/gdetrez/karma-webpack-issue/tree/b4fedcce44162716944b77622330552a5699d9f9

**Comment by nickpape-msft** [comment]
https://github.com/webpack-contrib/karma-webpack/pull/312

**Comment by michael-ciniawsky** [comment]
Released in [`v2.0.13`]https://github.com/webpack-contrib/karma-webpack/releases/tag/v2.0.13) 🎉
